apps/desktop-wallet/src/components/ActionLink.tsx

```diff
@@ -18,30 +18,49 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 
 import { colord } from 'colord'
 import { LucideIcon } from 'lucide-react'
+import { ReactNode } from 'react'
 import styled, { css } from 'styled-components'
 
 import { HasTooltip } from '@/components/Tooltip'
 
 interface ActionLinkProps {
+  children: ReactNode
   onClick: () => void
   Icon?: LucideIcon
   iconPosition?: 'right' | 'left'
   withBackground?: boolean
+  ellipsed?: boolean
   className?: string
 }
 
-const ActionLink: FC<HasTooltip<ActionLinkProps>> = ({ className, Icon, children, onClick, tooltip }) => (
-  <button className={className} onClick={onClick} data-tooltip-id="default" data-tooltip-content={tooltip}>
-    {children}
+const ActionLink = ({
+  className,
+  Icon,
+  children,
+  onClick,
+  ellipsed,
+  tooltip,
+  withBackground
+}: HasTooltip<ActionLinkProps>) => (
+  <ActionLinkStyled
+    className={className}
+    onClick={onClick}
+    data-tooltip-id="default"
+    data-tooltip-content={tooltip}
+    withBackground={withBackground}
+  >
+    <ChildrenContainer ellipsed={ellipsed}>{children}</ChildrenContainer>
```

Comment:
We currently have quite a few ways to truncate a text:
1. Using the `Ellipsed` component
2. Using the `Truncate` component 😆 
3. Using the `truncate` prop in `AddressBadge`

Let's first check if we can reuse some of them before introducing yet another prop to do the same thing, WDYT?

(and maybe also clean up a bit...)